server/utils/htmlSanitizer.js

```diff
@@ -17,12 +18,22 @@ function sanitize(html) {
 }
 module.exports.sanitize = sanitize
 
-function stripAllTags(html) {
+function stripAllTags(html, shouldDecodeEntities = true) {
   const sanitizerOptions = {
     allowedTags: [],
     disallowedTagsMode: 'discard'
   }
 
-  return sanitizeHtml(html, sanitizerOptions)
+  let sanitized = sanitizeHtml(html, sanitizerOptions)
+  return shouldDecodeEntities ? decodeHTMLEntities(sanitized) : sanitized
+}
+module.exports.stripAllTags = stripAllTags
+
+function decodeHTMLEntities(strToDecode) {
+  return strToDecode.replace(/\&([^;]+);/g, function (entity) {
```

Comment:
It looks like some of the entities don't end in a semicolon. At quick glance that is what this regex looks like it is matching for.

Reply:
Good catch.  Just pushed an update making it optional.